readiness-tool/popup.js

```diff
@@ -0,0 +1,218 @@
+/**
+ * @fileoverview Entry point for ART Chrome extension. 
+ * Scans the current page against the vendors in the app.json file and
+ * specifies them as either supported or not supported.
+ */
+self.popups = {};
+self.popups.isSupported = isSupported;
+self.popups.addToDict = addToDict;
+  /* @const {!Element} */
+let supportedAds;
+/* @const {!Element} */
+let supportedAnalytics;
+/* @const {!Element} */
+let notSupportedAds;
+/* @const {!Element} */
+let notSupportedAnalytics;
+/* @const {!Object} */
+let listAllApps;
```

Comment:
It's better to have globals than duplicating code, but it's even better to pass data around (functional programming style) rather than using globals.

Does this need to be a global?